src/Plugin/Field/FieldWidget/InlineEntityFormMultiple.php

```diff
@@ -240,7 +240,6 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
     $element['#element_validate'] = array('inline_entity_form_update_row_weights');
     // Add the required element marker & validation.
     if ($element['#required']) {
-      $element['#title'] .= ' ' . _theme('form_required_marker', array('element' => $element));
```

Comment:
I guess we shouldn't just delete this. Instead we need to change it to work with D8 correctly.